<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>530</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_sec">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_min">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="new_game_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>239</y>
      <width>63</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="sec_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sec</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
